web/docs/understand/operators/transformations/sort.md

```diff
@@ -1,43 +1,44 @@
 # sort
 
-Sort events.
-
-:::caution Not Yet Implemented
-The `sort` operator is still under development but will be available shortly.
-:::
+Sorts events.
 
 ## Synopsis
 
 ```
-sort <operand>...
+sort [--ascending|--descending] <field>
```

Comment:
I changed this to `sort <field> [<asc|desc>] [<nulls-first>|<nulls-last>]` since we talked this morning; I think we should agree on the exact syntax we want before merging this. What we have implemented now is my current preference.

Reply:
Okay, I've updated the docs again.

It looks like you haven't pushed your latest changes, though. Feel free to do so now.